crawl-ref/source/ability-type.h

```diff
@@ -82,7 +82,9 @@ enum ability_type
     ABIL_ZIN_VITALISATION,
     ABIL_ZIN_IMPRISON,
     ABIL_ZIN_SANCTUARY,
+#if TAG_MAJOR_VERSION == 34
     ABIL_ZIN_CURE_ALL_MUTATIONS,
```

Comment:
I'm probably missing something, but couldn't this ability just be removed entirely?  On that note, does this patch need to do something in tags.cc to fix up the abilities list?